### PR TITLE
Feature/build out user modal

### PR DESCRIPTION
**Pull request by jon-schlandt**
## Is this a fix or a feature?
- Feature

## What is the change?
- Built out the functionality for rendering user info onto the user info modal and opening/closing it
- This is currently going to be a strictly mobile feature

## Where should the reviewer start?
- With 'scripts.js' and 'domUpdates.js'

## How should this be tested?
- Access through browser, currently the app is only optimized for mobile

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h4>Date(s)</h4>
         <button id='userMenuBtn' class='user-menu-btn traveler'></button>
       </header>
       <section class='info-content'>
-        <h2>Hi, Travvler!</h2>
+        <h2 id='greeting'></h2>
```
Comment (Patfindley) [inline src/index.html:83]: good job at making this dynamic!

File: src/js/domUpdates.js [hunk 2]
```diff
@@ -62,6 +65,20 @@ export const domUpdates = {
     this.footer.classList.toggle('hidden');
     this.setAriaHidden(this.footer);
   },
+
+  renderUserInfo(info) {
+    let firstName;
+
+    if (info.name) {
```
Comment (Patfindley) [inline src/js/domUpdates.js:77]: nice error handling here

File: src/index.html [hunk 3]
```diff
@@ -80,7 +80,7 @@ <h4>Date(s)</h4>
...
         <div class='total-spent-sales'>
           <h4>Total Spent</h4>
           <p>$12,000</p>
```
Comment (Patfindley) [inline src/index.html:86]: is this cost hard-coded?

**Review by Patfindley — APPROVED** [review]

File: src/index.html (hunk 3 above)
Comment (jon-schlandt) [reply src/index.html:86]: Oh nice catch! It is, but it gets overwritten once I open the window its in, so the user never sees the original. 

I will definitely remove it though as it is unnecessary.

**Pull request merged**